examples/platform/silabs/shell/ble/BLEShellCommands.cpp

```diff
@@ -73,6 +72,24 @@ CHIP_ERROR StopBLESideChannelAvertising(int argc, char ** argv)
     return DeviceLayer::Internal::BLEMgrImpl().SideChannelStopAdvertising();
 }
 
+CHIP_ERROR NotifyBLESideChannel(int argc, char ** argv)
+{
+    CHIP_ERROR err = DeviceLayer::Internal::BLEMgrImpl().SideChannelNotify();
+    VerifyOrReturnError(err == CHIP_NO_ERROR, err,
+                        streamer_printf(streamer_get(), "Failed to notify BLE side channel: %s\n", ErrorStr(err)););
+    streamer_printf(streamer_get(), "Notified BLE side channel\n");
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR IndicateBLESideChannel(int argc, char ** argv)
+{
+    CHIP_ERROR err = DeviceLayer::Internal::BLEMgrImpl().SideChannelIndicate();
+    VerifyOrReturnError(err == CHIP_NO_ERROR, err,
+                        streamer_printf(streamer_get(), "Failed to indicate BLE side channel: %s\n", ErrorStr(err)););
+    streamer_printf(streamer_get(), "Indicated BLE side channel\n");
+    return CHIP_NO_ERROR;
+}
+
```

Comment:
Should all the handles also be in the namespace BLEShellCommands

Reply:
Sure